### PR TITLE
Ft account tests #147010551

### PR DESCRIPTION
**Pull request by margaretkinyanjui**
**What does this PR do?** 
the Accounts tests need fixing
 

**Description of Task to be completed?** 
Added login test and check token tests

**What are the relevant GitHub issues/pivotal tracker stories?** 
the Accounts tests need fixing

### CODE REVIEW

File: hc/accounts/tests/test_check_token.py [hunk 1]
```diff
@@ -18,11 +22,24 @@ def test_it_redirects(self):
...
+        
     ### Login and test it redirects already logged in
-
+    def test_loggedin_redirects(self):
+        user= User(username ='alice@example.org')
+        user.set_password('password')
+        user.save
+
+        self.client.login(username='alice@example.org',  password = 'password')
+        r = self.client.get("/")
```
Comment (bkiseu) [inline hc/accounts/tests/test_check_token.py]: name variables descriptively. PEP8 manenos

File: hc/accounts/tests/test_check_token.py [hunk 2]
```diff
@@ -18,11 +22,24 @@ def test_it_redirects(self):
...
+        self.assertRedirects(r, '/checks/')
     ### Login with a bad token and check that it redirects
+    
+    def test_login_badtoken(self):
+        
+        r = self.client.post("/accounts/check_token/alice/wrong-token/")
```
Comment (bkiseu) [inline hc/accounts/tests/test_check_token.py]: variable name not descriptive. Avoid single letter names

File: hc/accounts/tests/test_login.py [hunk 3]
```diff
@@ -33,5 +33,18 @@ def test_it_pops_bad_link_from_session(self):
...
+        check = Check()
+        check.save()
+
+        session = self.client.session
+        session["welcome_code"] = str(check.code)
+        session.save()
+
+        form = {"email": "alice@example.org"}
+
+        r= self.client.post("/accounts/login/", form)
```
Comment (bkiseu) [inline hc/accounts/tests/test_login.py]: variable name not cool

File: hc/accounts/tests/test_check_token.py [hunk 4]
```diff
@@ -18,11 +22,24 @@ def test_it_redirects(self):
         self.assertRedirects(r, "/checks/")
 
         # After login, token should be blank
-        self.profile.refresh_from_db()
+        self.profile.refresh_from_db() 
         self.assertEqual(self.profile.token, "")
-
+
```
Comment (andela-landia) [inline hc/accounts/tests/test_check_token.py:27]: remove new line

File: hc/accounts/tests/test_check_token.py [hunk 5]
```diff
@@ -18,11 +22,24 @@ def test_it_redirects(self):
...
     ### Login and test it redirects already logged in
-
+    def test_loggedin_redirects(self):
+        user= User(username ='alice@example.org')
+        user.set_password('password')
+        user.save
+
+        self.client.login(username='alice@example.org',  password = 'password')
+        r = self.client.get("/")
+        self.assertRedirects(r, '/checks/')
```
Comment (andela-landia) [inline hc/accounts/tests/test_check_token.py]: assert also that the status code is 301

File: hc/accounts/tests/test_login.py (hunk 3 above)
Comment (andela-landia) [reply hc/accounts/tests/test_login.py]: Consider naming of your variables

**Review by andela-landia — COMMENTED** [review]
Ensure you follow PEP8 standards

**Pull request merged**